src/kendo.calendar.js

```diff
@@ -950,7 +951,8 @@ var __meta__ = { // jshint ignore:line
                 today = new DATE(),
                 html = '<table tabindex="0" role="grid" class="k-content" cellspacing="0" data-start="' + toDateString(start) + '"><thead><tr role="row">';
                 if (isWeekColumnVisible) {
-                    html += '<th scope="col" class="k-alt"></th>';
+                    // an empty <th> is not 508 compliant and it should have some text content
+                    html += '<th scope="col" class="k-alt"><span style="visibility: hidden;">.</span></th>';

```

Comment:
@Xizario:
`<th scope="col">&nbsp;</th>` this is not working.
We had the same scenario for the grid and @Stamo-Gochev had some concerns for longer abbreviation than "."